.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is typically used to store user-uploaded files in Django projects. If you intend to keep these files out of version control, this is correct. However, ensure that this is intentional, as ignoring this directory will prevent any media files from being tracked by Git.